plugins/packages/polygon/main.go

```diff
@@ -100,3 +99,47 @@ func getBlockAuthor(header *evm.Header) (common.Address, error) {
 
 	return signer, nil
 }
+
+var (
+	polygonHitMeter  = metrics.NewMajorMeter("/flume/polygon/hit")
+	polygonMissMeter = metrics.NewMajorMeter("/flume/polygon/miss")
+)
+
+func borBlockDataPresent(input interface{}, cfg *config.Config, db *sql.DB) bool {
+	present := true
+	switch input.(type) {
+	case plugins.BlockNumber:
+		if uint64(input.(plugins.BlockNumber).Int64()) < cfg.EarliestBlock {
+			present = false
+			return present
+		}
+	case types.Hash:
+		blockHash := input.(types.Hash)
+		var response int
+		statement := "SELECT 1 FROM bor.bor_logs WHERE blockHash = ?;"
```

Comment:
I think we should bease this on blocks.blocks rather t han bor.bor_logs. Bor logs are only present on 1/64 blocks, so this would mean 63/64 blocks would get routed to Flume heavy even if Flume light has those blocks.